Makefile

```diff
@@ -50,6 +52,9 @@ test: build
 		--label build.buf.plugins.config.owner=$${PLUGIN_OWNER} \
 		--label build.buf.plugins.config.name=$${PLUGIN_NAME} \
 		--label build.buf.plugins.config.version=$${PLUGIN_VERSION} \
+		--label org.opencontainers.image.source=https://github.com/bufbuild/plugins \
+		--label "org.opencontainers.image.description=$${PLUGIN_DESCRIPTION}" \
+		--label "org.opencontainers.image.licenses=$${PLUGIN_LICENSE}" \
```

Comment:
Added recommended labels: https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry#labelling-container-images. This will only affect new/updated images (not planning to rebuild and republish everything to add these labels).